apps/nirvlife/src/Components/App/AppLandingScreen.tsx

```diff
@@ -2,17 +2,22 @@ import { useTheme } from "@mui/material";
 
 import { Img, TextCopy } from "Library";
 import { ctaStacks, nirvaiRef, taglineRef } from "data";
-import { ReactFP, FPContainer, FPItem } from "react-fullerpage";
-// <img src={"/img/logo_main.png"} className="img" alt="nirvai" />
+// import { ReactFP, FPContainer, FPItem } from "react-fullerpage";
+import {
```

Comment:
use the main package as we can now develop with react-cosmos and dont need to copypaste into node_modules

also check on the bun issue with bun link not working